mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/Utils.kt

```diff
@@ -0,0 +1,27 @@
+package ch.epfl.sdp.mobile.test.state
+
+import android.app.Activity
+import android.app.Instrumentation.ActivityResult
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+
+/** Object that contains helper function that will be used in tests */
+object Utils {
+  /**
+   * Executes the given [block] by returning an [ActivityResult] with the code
+   * [Activity.RESULT_CANCELED] each time an intent is triggered. This simulates devices which do
+   * not support AR.
+   *
+   * @param block the block of code to execute.
+   */
+  fun withCanceledIntents(block: () -> Unit) {
```

Comment:
You may want to make this function top-level rather than in an a `Utils` `object` (having a dedicated object for utility functions isn't the norm in Kotlin; the only reason for which you'd do this was if the object served the purpose of being a namespace with a restricted domain, which _Utils_ isn't).

Reply:
Fix in commit https://github.com/epfl-SDP/android/pull/251/commits/2aad493f16e046f6b82dfd7e89a87f0d39691b6a